### PR TITLE
Activate pycodestyle test in travis (and fix some of the errors)

### LINKED ISSUE #446: PEP8 travis check
Would it be worthwhile to enable the `pycodestyle` (PEP8) Travis job? Like [the one in astropy.](https://github.com/astropy/astropy/blob/master/.travis.yml#L105)?

### PR DESCRIPTION
**Pull request by MSeifert04**
Fixes #446.

Let me know if I should include additional tests or remove some of these.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -72,14 +72,20 @@ matrix:
         - python: 3.6
           env: NUMPY_VERSION=prerelease
 
+        # pycodestyle test, see setup.cfg for tested inconsistencies
+        - python: 3.6
+          env: MAIN_CMD='pycodestyle ccdproc --count  --show-source --show-pep8'
+               SETUP_CMD=''
+               CONDA_DEPENDENCIES='pycodestyle'
```
Comment (bsipocz) [inline .travis.yml]: you don't need this, ``ci-helpers`` checks for ``'pycodestyle'`` in ``MAIN_CMD`` and installs it when needed.

File: setup.cfg [hunk 2]
```diff
@@ -26,4 +26,42 @@ url = http://ccdproc.readthedocs.io/
...
+# W391 - blank line at end of file
+
+[pycodestyle]
+select = E101,E111,E112,E113,E221,E222,E223,E224,E225,E241,E242,E251,E271,E272,E303,E304,E502,E703,E901,E902,W191,W291,W292,W293,W391
+
+# E402 module level import not at top of file
+# We need to ignore this as the .conf has to be set in the __init__.py
+# before the modules are imported.
+ignore = E402
+
```
Comment (bsipocz) [inline setup.cfg]: you may want to add an exclude list here for files that are generated or pure copies from the template (and fix issues in the template instead). For astroquery I currently have this:
```
# Excluding files that are directly copied from the package template or
# generated
exclude = _astropy_init.py,version.py
```

File: setup.cfg [hunk 3]
```diff
@@ -26,4 +26,42 @@ url = http://ccdproc.readthedocs.io/
...
+# W293 - blank line contains whitespace
+# W391 - blank line at end of file
+
+[pycodestyle]
+select = E101,E111,E112,E113,E221,E222,E223,E224,E225,E241,E242,E251,E271,E272,E303,E304,E502,E703,E901,E902,W191,W291,W292,W293,W391
+
+# E402 module level import not at top of file
+# We need to ignore this as the .conf has to be set in the __init__.py
+# before the modules are imported.
+ignore = E402
```
Comment (bsipocz) [inline setup.cfg]: How often do you have this in ccdproc? We have many configs in astroquery, this it's always ignored, but you may not want to copy that.

**Comment by MSeifert04** [comment]
@bsipocz Thanks for your comments! 👍

**Comment by crawfordsm** [comment]
This looks good to me -- I think it is a reasonable set of tests being used, and we can adjust them if we find anything problematic or if it interferes too much.  

Merge when you are happy with it @mwcraig !

**Pull request merged**